### PR TITLE
ui: Flamegraph tooltip improvements

### PR DESCRIPTION
**Pull request by manojVivek**

### CODE REVIEW

File: ui/packages/shared/profile/src/utils.ts [hunk 1]
```diff
@@ -50,3 +50,11 @@ export const truncateString = (str: string, num: number): string => {
 
   return str.slice(0, num) + '...';
 };
+
+export const truncateStringReverse = (str: string, num: number): string => {
```
Comment (monicawoj) [inline ui/packages/shared/profile/src/utils.ts:54]: I think this is a good example of something that could live in the @parca/utils package once that is in place :) cc @yomete

**Comment by monicawoj** [comment]
In general, looks great to me! 

My only concern is the empty whitespace in the tooltip when hovering over the root node (or a node with very little information to show).. I don't assume the user will spending much time hovering over root, but just wanted to bring it up and get some others' thoughts on this.
Screenshot 2023-02-15 at 22 19 33

**Comment by manojVivek** [comment]
> My only concern is the empty whitespace in the tooltip when hovering over the root node (or a node with very little information to show).. I don't assume the user will spending much time hovering over root, but just wanted to bring it up and get some others' thoughts on this.

Yes, I had the concern as well. Should we try listing all fields everywhere and mark that there's `no data` wherever we don't have the info? 

@brancz Let me know what do you think as well.

**Comment by brancz** [comment]
> Yes, I had the concern as well. Should we try listing all fields everywhere and mark that there's no data wherever we don't have the info?

I think this is a good idea.

**Review by yomete — APPROVED** [review]

**Pull request merged**